Sources/Instrumentation/InstrumentationSystem.swift

```diff
@@ -14,13 +14,24 @@
 import Baggage
 
 /// `InstrumentationSystem` is a global facility where the default cross-cutting tool can be configured.
-/// It is set up just once in a given program select the desired `Instrument` implementation.
+/// It is set up just once in a given program to select the desired `Instrument` implementation.
 ///
-/// - Note: If you need to use more that one cross-cutting tool you can do so by using `MultiplexInstrument`.
+/// # Bootstrap multiple Instruments
+/// If you need to use more that one cross-cutting tool you can do so by using `MultiplexInstrument`.
+///
+/// # Access the Instrument
+/// There are two ways of getting the bootstrapped instrument.
+/// 1. `InstrumentationSystem.instrument`: Returns whatever you passed to `.bootstrap` as an `Instrument`.
+/// 2. `InstrumentationSystem.instrument(of: MyInstrument.self)`: Returns the bootstrapped `Instrument` if it's
+/// an instance of the given type or the first instance of `MyInstrument` if it's part of a `MultiplexInstrument`.
+///
+/// ## What getter to use
+/// - Default to using `InstrumentationSystem.instrument`
+/// - Use `InstrumentationSystem.instrument(of: MyInstrument.self)` only if you need to use specific `MyInstrument` APIs
+/// - Protocols that "inherit" from `Instrument` may also wrap `.instrument(of: TheirInstrument.self)` in a convenience method
```

Comment:
Can you explain a bit more what you mean here?

Do we mean here that libs should provide their own `extension InstrumentationSystem { var coolTracer }`? If so I didn't  understannd it this way hmm

Reply:
Yes, wanted to describe exactly that. Did you understand something else instead?

Reply:
After re reading it a few times I see what it desccribes but yeah it's a bit confusingly worded, maybe we can improve a but annd make it more example-ish.

```suggestion
/// - Specific instrumentation libraries may also provide their own accessors as extensions, e.g. GreatInstrumentation could provide a `InstrumentationSystem.great` convenience accessor, so prefer using them if available
```

Reply:
Looks good 👍 I'd like to add a sentence for implementors to note that they should use `instrument(of:)` when implementing such convenience methods. What do you think?

```suggestion
/// - Specific instrumentation libraries may also provide their own accessors as extensions, e.g. GreatInstrumentation could provide a `InstrumentationSystem.great` convenience accessor, so prefer using them if available. These accessors should call `.instrument(of: GreatInstrument.self)` under the hood to ensure they work when using a `MultiplexInstrument` as well.
```

Reply:
Sounds good 👍